package.json

```diff
@@ -49,12 +68,24 @@
     "koa-mount": "2.0.0",
     "koa-router": "^7.0.1",
     "koa-send": "^4.1.0",
+    "koa-simple-healthcheck": "^0.0.1",
+    "load-grunt-tasks": "^5.1.0",
```

Comment:
this is a dev dependency

Reply:
Eslint complaied that this should be a project dependency, not a dev:
![image](https://user-images.githubusercontent.com/2981676/70428381-674c4d80-1a6e-11ea-89b5-e843a985890a.png)

Anyway, I reset it as dev dependency!

Reply:
https://github.com/resource-watch/dataset/blob/08f08424362866dc8eb5c479b958ad93afc88c7c/app/Gruntfile.js#L4